Vagrantfile

```diff
@@ -0,0 +1,123 @@
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  config.vm.synced_folder ".", "/opt/src/vector-datasource"
+  config.vm.synced_folder "../tilequeue", "/opt/src/tilequeue"
+  config.vm.synced_folder "../tileserver", "/opt/src/tileserver"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+  #   # Customize the amount of memory on the VM:
+    vb.memory = "2048"
+  end
+  #
+  # View the documentation for the provider you are using for more
+  # information on available options.
+
+  # Define a Vagrant Push strategy for pushing to Atlas. Other push strategies
+  # such as FTP and Heroku are also available. See the documentation at
+  # https://docs.vagrantup.com/v2/push/atlas.html for more information.
+  # config.push.define "atlas" do |push|
+  #   push.app = "YOUR_ATLAS_USERNAME/YOUR_APPLICATION_NAME"
+  # end
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  config.vm.provision "shell", inline: <<-SHELL
+    set -x
+    set -e
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+    export DEBIAN_FRONTEND=noninteractive
+    apt-add-repository -y ppa:tilezen
+    apt-get -qq update
+    apt-get -qq install -y git unzip \
+        postgresql postgresql-contrib postgis postgresql-9.3-postgis-2.1 \
+        build-essential autoconf libtool pkg-config \
+        python-dev python-virtualenv libgeos-dev libpq-dev python-pip python-pil libmapnik2.2 libmapnik-dev mapnik-utils python-mapnik \
+        osm2pgsql
+
+    sudo -u postgres psql -c "CREATE ROLE osm WITH NOSUPERUSER LOGIN UNENCRYPTED PASSWORD 'osmpassword';"
+    sudo -u postgres createdb -E UTF-8 -T template0 -O osm osm
+    sudo -u postgres psql -d osm -c 'CREATE EXTENSION postgis; CREATE EXTENSION hstore;'
+
+    export PGPASSWORD=osmpassword
+
+    mkdir -p /opt/data /opt/src
+    chown vagrant:vagrant -R /opt
+    su vagrant
+
+    # Download and import OSM data
+    wget -q -P /opt/data/ https://s3.amazonaws.com/metro-extracts.mapzen.com/new-york_new-york.osm.pbf
```

Comment:
Yep. The `-q` is equivalent to `--quiet`. I'll switch to the longer version to make it clearer.